requirements.txt

```diff
@@ -0,0 +1,4 @@
+setuptools
+networkx
+Cython
+PyQt5==5.14
```

Comment:
Is there a specific reason for pinning this version?

Reply:
`PyQt5` latest version (5.14.1) fails to install for some reason as mentioned on this stackoverflow thread:
https://stackoverflow.com/questions/59711301/install-pyqt5-5-14-1-on-linux

Reply:
In that case, you should handle this in the Dockerfile. If there is a `requirements.txt` then it should reflect the same version dependencies as stated in the README.